src/NuGet.Core/NuGet.Commands/Strings.resx

```diff
@@ -1102,4 +1102,7 @@ For more information, visit https://docs.nuget.org/docs/reference/command-line-r
     <value>The project {0} is using CentralPackageVersionManagement, a NuGet preview feature.</value>
     <comment>{0} the project path.</comment>
   </data>
+  <data name="Error_CentralPackageVersions_MissingPackageVersion" xml:space="preserve">
+    <value>The PackageReference items {0} do not have corresponding PackageVersions.</value>
```

Comment:
The item name is `PackageVersion` right? 

We should not pluralize the MSBuild item names.

Reply:
Removed the "s".